src/otserv.cpp

```diff
@@ -324,3 +331,47 @@ void mainLoader(int, char*[], ServiceManager* services)
 	g_game.setGameState(GAME_STATE_NORMAL);
 	g_loaderSignal.notify_all();
 }
+
+bool argumentsHandler(const StringVector& args)
+{
+	for (const auto& arg : args) {
+		if (arg == "--help") {
+			std::clog << "Usage:\n"
+			"\n"
+			"\t--config=$1\t\tAlternate configuration file path.\n"
+			"\t--ip=$1\t\t\tIP address of the server.\n"
+			"\t\t\t\tShould be equal to the global IP.\n"
+			"\t--login-port=$1\tPort for login server to listen on.\n"
+			"\t--game-port=$1\tPort for game server to listen on.\n";
+			return false;
+		} else if (arg == "--version") {
+			std::cout << STATUS_SERVER_NAME << " - Version " << STATUS_SERVER_VERSION << std::endl;
+			std::cout << "Compiled with " << BOOST_COMPILER << std::endl;
+			std::cout << "Compiled on " << __DATE__ << ' ' << __TIME__ << " for platform ";
+
+#if defined(__amd64__) || defined(_M_X64)
+			std::cout << "x64" << std::endl;
+#elif defined(__i386__) || defined(_M_IX86) || defined(_X86_)
+			std::cout << "x86" << std::endl;
+#elif defined(__arm__)
+			std::cout << "ARM" << std::endl;
+#else
+			std::cout << "unknown" << std::endl;
+#endif
```

Comment:
This doesn't match what we have in otserv.cpp now. Maybe it shouldn't be copy-pasta'd, but a function somewhere.